.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,17 +2,20 @@
 name: Specified Feature
 about: Agreed specification for a feature or enhancement
 title: ''
-labels: 'feature'
+labels: 'feature request'
 assignees: ''
 ---
 
-<!-- Thanks for specifying a new feature! -->
+**Provide context**
+<!-- What's the status quo and how is this non-optimal?** -->
 
-**Has this feature been discussed and generally agreed?**
-<!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed specification such that anyone could action it without much help. -->
+**Describe possible approaches**
```

Comment:
I think the user should also be lead to think about KISS. Can this be accieved by another way? Another software? Why should this be supported by Jamulus?
Comment: Please note the contribution guidelines (link here), especially that new features should be added with caution to keep Jamulus simple.

Reply:
Updated. Can you re-check?

I agree with all of your points, but I think it has downsides to have rather long explanations, too. People may either ignore it or walk away with some "too complicated" feeling.

But as I mainly care about the order, I've added your suggestions.

Reply:
Yes, thought of that too. Let me think about something else...